Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -703,7 +704,7 @@ bool AccessibilityNodeObject::computeIsIgnored() const
     if (node->isTextNode() && !renderer()) {
         RefPtr parent = node->parentNode();
         // Fallback content in iframe nodes should be ignored.
-        if (parent && parent->hasTagName(iframeTag) && parent->renderer())
+        if (parent && WebCore::elementName(parent.get()) == ElementName::HTML_iframe && parent->renderer())
```

Comment:
Could use the Node& version of `elementName` to avoid duplicating the null-check.